mysql/README.md

```diff
@@ -186,9 +186,13 @@ primary key columns to be `NOT NULL`. However, in MySQL, a primary key is a
 combination of `NOT NULL` and `UNIQUE`, and so primary key columns from
 MySQL will be mapped to Spanner columns that are both primary keys and `NOT NULL`.
 
-### Foreign Keys and Default Values
+### Foreign Keys
 
-Spanner does not currently support foreign keys or default values. We drop these
+Spanner doesn't support `ON DELETE` and `ON UPDATE` rules in foreign keys, so we skip those rules. We handle foreign keys using ALTER TABLE statement after all the tables are created, this helps us avoid cyclic dependencies in foreign keys.
```

Comment:
Suggest rephrasing:

  The tool maps MySQL foreign key constraints into Spanner foreign key constraints, and
  preserves constraints names where possible. Since Spanner doesn't  support `ON DELETE`
  and `ON UPDATE` actions, we drop them.

I'd drop the comment about using ALTER TABLE: I don't think it is needed (it's kind of an implementation detail), and in any case Postgre and MySQL dump formats do essentially the same thing so it's not a surprise.